src/scan.l

```diff
@@ -682,7 +682,19 @@ F	RET_CONST(zeek::val_mgr->False()->Ref())
 
 "0x"{HEX}+	RET_CONST(zeek::val_mgr->Count(static_cast<zeek_uint_t>(strtoull(yytext, 0, 16))).release())
 
-({H}".")+{HTLD}		RET_CONST(zeek::detail::dns_mgr->LookupHost(yytext).release())
+({H}".")+{HTLD} {
+	zeek::TableValPtr result = zeek::detail::dns_mgr->empty_addr_set();
+
+	if ( ! zeek::detail::parse_only ) {
```

Comment:
Any reason we suppress the error with `--parse-only`? I used `--parse-only` for very basic linting in some editor integrations, and it would be great if we could surface it in that mode, before deployment.

Reply:
Hmm, fair. If one actually wants to use the feature, they'll need to inject the env variable into their development environment, but maybe that's their issue then.

This also behaves different with `--parse-only` now - though I don't think I care all that much :smiling_face_with_tear: 
```
@if ( |zeek.org| > 0 )
        print 1;
@endif
```

Let me ditch the parse_only special case!